src/components/LoanCards/Buttons/LendCtaExp.vue

```diff
@@ -0,0 +1,339 @@
+					<kv-ui-select
+						v-if="hideShowLendDropdown && !isLessThan25"
+						id="LoanAmountDropdown"
+						class="tw-min-w-12"
+						v-model="selectedOption"
+					>
+						<option
+							v-for="priceOption in prices"
+							:key="priceOption"
+							:value="priceOption"
+						>
+							${{ priceOption }}
+						</option>
+					</kv-ui-select>
+				</div>
+
+				<!-- Lend button -->
+				<div :class="{ 'lendButtonWrapper' : hideShowLendDropdown}">
+					<kv-ui-button
+						key="lendButton"
+						v-if="lendButtonVisibility && !isLessThan25"
+						class="tw-inline-flex tw-flex-1"
+						data-testid="bp-lend-cta-lend-button"
+						type="submit"
+					>
+						{{ ctaButtonText }}
+					</kv-ui-button>
+				</div>
+
+				<!-- Lend again/lent previously button -->
+				<kv-ui-button
+					key="lendAgainButton"
+					v-if="isLentTo && !isLessThan25"
+					class="tw-inline-flex tw-flex-1"
+					data-testid="bp-lend-cta-lend-again-button"
+					type="submit"
+				>
+					Lend again
+				</kv-ui-button>
+
+				<!-- Stranded loans -->
+				<lend-amount-button
+					class="tw-w-full"
+					:loan-id="loanId"
+					:show-now="false"
+					:amount-left="unreservedAmount"
+					@add-to-basket="addToBasket"
+					v-if="isLendAmountButton"
+				/>
+
+				<!-- Adding to basket button -->
+				<kv-ui-button
+					v-if="isAdding"
+					class="tw-inline-flex tw-flex-1"
+				>
+					Adding to basket
+				</kv-ui-button>
+			</fieldset>
+		</form>
+
+		<!-- Continue to checkout button -->
+		<kv-ui-button
+			v-if="this.state === 'basketed'"
+			variant="secondary"
+			class="tw-inline-flex tw-flex-1"
+			data-testid="bp-lend-cta-checkout-button"
+			to="/basket"
+		>
+			Checkout now
+		</kv-ui-button>
+
+		<!-- Funded, refunded, expired/ allSharesReserved button -->
+		<kv-ui-button
+			v-if="showNonActionableLoanButton"
+			class="tw-inline-flex tw-flex-1"
+		>
+			{{ ctaButtonText }}
+		</kv-ui-button>
+	</div>
+</template>
+
+<script>
+import { gql } from '@apollo/client';
+import { setLendAmount } from '@/util/basketUtils';
+import {
+	buildPriceArray,
+	isLessThan25,
+	isBetween25And50,
+	isBetween25And500
+} from '@/util/loanUtils';
+
+import LendAmountButton from '@/components/LoanCards/Buttons/LendAmountButton';
+
+import KvUiSelect from '~/@kiva/kv-components/vue/KvSelect';
+import KvUiButton from '~/@kiva/kv-components/vue/KvButton';
+
+export default {
+	name: 'LendCtaExp',
+	inject: ['apollo', 'cookieStore'],
+	props: {
+		loanId: {
+			type: Number,
+			default: 0,
+		},
+	},
+	components: {
+		LendAmountButton,
+		KvUiButton,
+		KvUiSelect,
+	},
+	data() {
+		return {
+			selectedOption: '25',
+			loanAmount: '',
+			fundedAmount: '',
+			reservedAmount: '',
+			unreservedAmount: '',
+			lentPreviously: false,
+			minNoteSize: '',
+			status: '',
+			basketItems: [],
+			isAdding: false,
+			isLoading: true,
+			completeLoanView: true,
+		};
+	},
+	apollo: {
+		query: gql`
+			query lendCta($loanId: Int!, $basketId: String) {
+				lend {
```

Comment:
Is there a reason that the CTA needs to query all of the loan data again? It looks like the `KivaClassicBasicLoanCardExp` `loan` data has all of the needed props.

Reply:
I see, lemme update it to pass loan data from parent

Reply:
done

Reply:
Lemme update the stories